cadccutout/cadccutout/version.py

```diff
@@ -1 +1 @@
-version = '0.1.0'
+version = '0.1.1'
```

Comment:
Sorry I didn't catch this initially. The file is not in version control. It should be dynamically generated when package installed. Please remove from git.